src/main/py/trade/trade_confirmation.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+"""
+Module to manage trade confirmations.
+
+TradeConfirmation class allows us to respond to a trade responses
+"""
+
+
+class TradeConfirmation:
```

Comment:
This is another case (like [mentioned here](https://github.com/zimmermatt/ob-winterm24-p2p/pull/30#discussion_r1468277645) in PR #30) that would be perfect as a @dataclass.

See https://docs.python.org/3/library/dataclasses.html